mkdocs.yml

```diff
@@ -61,6 +61,7 @@ nav:
     - About Orchard Core: docs/README.md
     - Getting started: 
         - Create a CMS Web application: docs/getting-started/README.md
+        - Configure Dev package source: docs/getting-started/devPackageSource.md
```

Comment:
If we're using dashes everywhere then this should also rather be dev-package-source.md.

Reply:
ok